.github/workflows/ci.yml

```diff
@@ -464,6 +464,69 @@ jobs:
       # TODO: Do this check on Windows too.
       - run: mk/check-symbol-prefixes.sh --target=${{ matrix.target }}
 
+  # The wasm32-wasi targets have a different set of feature sets.
```

Comment:
I do not understand this comment about feature sets, and thus I don't understand why the wasi target needs its own build matrix.